hardhat.config.ts

```diff
@@ -1,13 +1,30 @@
-import '@typechain/hardhat'
-import '@nomiclabs/hardhat-ethers'
-import '@nomiclabs/hardhat-waffle'
+import '@typechain/hardhat';
+import '@nomiclabs/hardhat-ethers';
+import '@nomiclabs/hardhat-waffle';
+
+// THIS PRIVATE KEY SHOULD ONLY BE USED IN TEST ENVIRONMENTS
+// It has been leaked publicly and its funds can be stolen at any time
+// Replace with your own private key
+const deployKey = "0xd9066ff9f753a1898709b568119055660a77d9aae4d7a4ad677b8fb3d2a571e5";
 
 // noinspection JSValidateJSDoc
 /**
  * @type import('hardhat/config').HardhatUserConfig
  */
 module.exports = {
   solidity: "0.8.4",
+  networks: {
+    volta: {
+      url: "https://volta-rpc.energyweb.org",
+      accounts: [deployKey],
+      chainId: 73799
+    },
+    ewc: {
+      url: "https://rpc.energyweb.org",
+      accounts: [deployKey],
```

Comment:
an alternative to the above comment would be to import the key only for EWC deployments by renaming the `deployKey` to something like `deployKeyEWC` and importing this one from an external file if it exists